src/nomad_material_processing/general.py

```diff
@@ -545,24 +580,41 @@ class SubstrateCrystalProperties(CrystalProperties):
             component=ELNComponentEnum.EnumEditQuantity,
         ),
     )
-    orientation = Quantity(
+    hkl_reciprocal = Quantity(
         type=str,
-        description="""
-        Alignment of crystal lattice with respect to a vector normal to the surface
-        specified using Miller indices.
-        """,
+        description="""The reciprocal lattice vector associated with the family of
+        lattice planes is OH = h a* + k b* + l c*, where a*, b*, c* are the reciprocal
+        lattice basis vectors. OH is perpendicular to the family of lattice planes and
+        OH = 1/d where d is the lattice spacing of the family.""",
         a_eln=ELNAnnotation(
             component=ELNComponentEnum.StringEditQuantity,
-            label='Substrate Orientation (hkl)',
+            label='Reciprocal Indices (hkl)',
+        ),
+    )
+    hkl_direct = Quantity(
+        type=str,
+        description="""In three-dimensional space, the direction passing through the
+        origin and the lattice nodes nh,nk,nl, where n is an integer, has direction
+        indices [hkl]. This corresponds to taking the coordinates of the first lattice
+        node on that direction after the origin as direction indices.
+        When a primitive unit cell is used, the direction indices are all integer;
+        they may instead be rational when a centred unit cell is adopted.""",
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+            label='Direct Orientation [hkl]',
```

Comment:
I think we need to somehow clarify that these are the directions of the surface.